bruteforce/greedy/4796.캠핑.kt

```diff
@@ -0,0 +1,23 @@
+import java.io.BufferedReader
+import java.io.BufferedWriter
+import java.io.InputStreamReader
+import java.io.OutputStreamWriter
+import java.lang.Integer.min
+
+val br = BufferedReader(InputStreamReader(System.`in`))
+val bw = BufferedWriter(OutputStreamWriter(System.out))
+fun main() {
+    with(br) {
+        var cnt = 0
+        while (true) {
+            cnt++
+            val list: List<Int> = readLine().split(" ").map { it.toInt() }

```

Comment:
```suggestion
            val (l, p, v) = readLine().split(" ").map { it.toInt() }
```
이미 알고계실 것 같은데 위와 같이 코드를 작성하면 리스트 타입이 아닌 int타입 그대로 사용할 수도 있을 것 같습니다! 고럼 인덱스로 리스트에 접근할 필요도 없겠죠?!

Reply:
하하.. 사실 코드를 치면서, 그냥 변수에 담을껄~ 이라고 생각은 들었지만 ~~귀찮아서~~ 내버렸답니당 ㅋㅋㅋㅋ